code/modules/clothing/spacesuits/syndi.dm

```diff
@@ -4,7 +4,7 @@
 	desc = "Top secret Spess Helmet."
 	icon_state = "syndicate"
 	item_state = "syndicate"
-	desc = "Has a tag: Totally not property of an enemy corporation, honest."
+	desc = "Has a tag on it: Totally not property of of a hostile corporation, honest!"

```

Comment:
check the master branch, that was there for the softsuit description already, I (and everyone else) just didn't notice it..